scripts/validate_string_concatenation.py

```diff
@@ -1,51 +1,39 @@
 #!/usr/bin/env python
 """
-GH #30454
-
-Check where there is a string that needs to be concatenated.
-
-This is necessary after black formating,
-where for example black transforms this:
-
->>> foo = (
-...         "bar "
-...         "baz"
-...     )
-
-into this:
-
->>> foo = ("bar " "baz")
-
-Black is not considering this as an
-issue (see issue https://github.com/psf/black/issues/1051),
-so we are checking it here.
+Unwanted patterns test cases.
 """
 
 import argparse
 import os
 import sys
 import token
 import tokenize
-from typing import Generator, List, Tuple
+from typing import Callable, Generator, List, Tuple
 
 FILE_EXTENSIONS_TO_CHECK = (".py", ".pyx", ".pyx.ini", ".pxd")
 
 
-def main(source_path: str, output_format: str) -> bool:
+def main(
```

Comment:
Not sure if it's just my own convention, but I'd say you usually sort functions in the opposite order. So the last one is the `if __name__...` block, the previous is `main`, and the first functions are the ones that don't call anything from this function.

The idea is that when a function is being defined, the functions that it uses are already defined (in compiled languages things can fail otherwise).